client/luigi-client-wc-docu-mixin.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * <!-- label-success: Web Component API -->
+ * Publish an event that can be listened to from the container host.
+ *
+ * Similar to {@link luigi-client-api.md#sendCustomMessage sendCustomMessage} but for WebComponent based microfrontends only.
+ *
+ * @param {CustomEvent} event Custom event to be published
+ * @memberof Lifecycle
+ *
+ * @example
+ * ** WC Container Scenario **
+ * Sending a message from a WC based microfrontend to parent host
+ *
+ * // wcComponent.js
+ * this.LuigiClient.publishEvent(new CustomEvent('sendSomeMsg', { detail: 'My own message' }));
+ *
+ * // host.html
+ * myContainer.addEventListener('custom-message', event => {
+ *   console.log('My custom message from microfrontend', event.detail.data);
+ * }
+ *
+ * ** Compound Container Scenario **
+ *
+ * // Sending a message from child compound microfrontend (secondChild.js) to its parent (main.html)
+ * // and siblings (firstChild.js) through the event bus
+ *
+ * // secondChild.js
+ * // Set the custom event name = sendInput
+ * this.LuigiClient.publishEvent(new CustomEvent('sendInput', { detail: 'My own message' }));
+ *
+ * // main.html
+ * myContainer.addEventListener('custom-message', event => {
+ *   console.log('My custom message from microfrontend', event.detail.data);
+ * }
+ *
+ * // *Note:* eventListeners.name must match CustomEvent name above
```

Comment:
could we make this render as a note like in a >**Note** with the comment as the text and the config in a codeblock inside the Note?

Reply:
ah, I checked and I think the way its structured, it seems like all content in Examples: shows as a one-piece code example.